src/actions.tex

```diff
@@ -615,10 +596,19 @@
 	\item Cath. Realms have Holy War CB against Muslim Realms Owning Prov. in the Area	
 \end{itemize}
 
+\subsectionstriped{colorHre}{HRE Actions}
+\actionHeadingNoSpace{Increase \authority \normal{(1\adminpower + current \authority) (p.~43)}}
+\begin{itemize}
+	\item You must be the Emperor
+	\item Increase \authority by 1
+\end{itemize}
+
 \subsectionstriped{colorAdmin}{Admin. Actions}
 \actionHeadingNoSpace{Incr. Stab. \normal{(5\adminpower ± current \stability) (p.~14)}}
 \begin{itemize}
 	\item Increase \stability by 1
+	\item If Papal Controller, pay -1\adminpower (p.~46)
+	\item If Orthodox and current \stability is -1/0, pay -1\adminpower (p.~38)
```

Comment:
I was also thinking that it might be better to write `1\adminpower less` instead of -1 in places like this (when there is enough room)

Reply:
I added the change where possible in the actions page.
While checking the whole document for this, I spotted some phrasing improvements in the sequence of play. There I left the "receive +/-X" form as it seemed clearer, but I added a couple of details. Let me know if it is ok to squeeze in past reviewed files changes like this if it happens.

Reply:
> I added the change where possible in the actions page. While checking the whole document for this, I spotted some phrasing improvements in the sequence of play. There I left the "receive +/-X" form as it seemed clearer, but I added a couple of details. Let me know if it is ok to squeeze in past reviewed files changes like this if it happens.

Yes, no problem. In case of bigger changes like moving boxes around I'd prefer a separate PR, though